scripts/travis-emscripten/install_deps.sh

```diff
@@ -35,6 +35,8 @@ wget 'https://sourceforge.net/projects/boost/files/boost/1.67.0/boost_1_67_0.tar
  -O - | tar xz
 cd boost_1_67_0
 ./bootstrap.sh --with-toolset=gcc
+wget 'https://raw.githubusercontent.com/tee3/boost-build-emscripten/master/emscripten.jam'
```

Comment:
If we wget this we must add a hash verification here, otherwise this is a security risk on the release process.

Reply:
Strange, why don't we do the hash verification on the `boost.tar.gz`?!

Reply:
I would even consider adding the file to our repo. It's licensed under the boost license, so it shouldn't be a problem? Not sure how "official" that tee3 repository is (although they have an issue for submitting for inclusion in official boost https://github.com/tee3/boost-build-emscripten/issues/3). It was just the first one I found that seemed to work. Although it might help push inclusion in official boost if we ``wget`` and hash it instead...

I'll look into the hash verification of ``boost`` as well - I hadn't thought about it, but it would definitely be good to have, if I didn't just disable it for this PR.